src/components/Header/NavBar.js

```diff
@@ -2,36 +2,45 @@ import Link from 'next/link';
 import Head from 'next/head';
 import Image from 'next/image';
 import styles from './NavBar.module.css';
+import ReusableNavLink from '../common/ReusableNavLink';
+
 
 export default function NavBar() {
+    const navList = [
+        { text: "About", href: "/about" },
+        { text: "Contact", href: "/contact" },
+      ];
+
     return (
-        <div className={styles.container}>
-            <Image
-                src="/vercel.svg"
-                alt="Vercel Logo"
-                className={styles.vercelLogo}
-                width={100}
-                height={24}
-                priority
-              />
-            
-            <form>
-                <input id="searchInputBar" type="text" placeholder="Search" name='search' required/>
-                <button type='submit'>
-                    <Image
-                        src="/search-glass.svg"
-                        alt="Search Image"
-                        className={styles.searchImage}
-                        width={20}
-                        height={20}
-                    />
-                </button>
-            </form>
-            
-            <ul className={styles.nav}>
-                <li>Create Listing</li>
-                <li>Log Out</li>
+        <div className=" relative flex items-center justify-evenly place-content-center sm:text-left md:flex py-4 px-24 lg:flex-row flex-col">
```

Comment:
@istoga Good job so far, but some changes needs to be made

- In line 15 div should  justify items with space between, place-content-center is redundant here, reduce the padding left and right less than 10(px)
-

Reply:
Done